services/scan-server/src/main/java/org/csstudio/scan/server/internal/JythonSupport.java

```diff
@@ -174,15 +174,17 @@ public JythonSupport() throws Exception
         }
         catch (PyException ex)
         {
-            logger.log(Level.WARNING, "Error loading Jython class {0} from {1}",
-                new Object[] { class_name, pack_name });
-            logger.log(Level.WARNING, "Jython sys.path:\n * {0}",
-                       interpreter.getSystemState()
-                                  .path
-                                  .stream()
-                                  .collect(Collectors.joining("\n * ")));
-
-            throw new Exception("Error loading Jython class " + class_name + ":" + getExceptionMessage(ex), ex);
+            try (final var state = interpreter.getSystemState()) {
```

Comment:
`interpreter.getSystemState()` looks like this:
```
    public PySystemState getSystemState() {
        return systemState;
    }
```

It returns a reference to the shared `systemState` which you must not `close()`.